src/nvim/macros.h

```diff
@@ -154,6 +154,6 @@
 /// zero in those cases (-Wdiv-by-zero in GCC).
 #define ARRAY_SIZE(arr) ((sizeof(arr)/sizeof((arr)[0])) / ((size_t)(!(sizeof(arr) % sizeof((arr)[0])))))
 
-#define RGB(r, g, b) ((r << 16) | (g << 8) | b)
+#define NVIM_RGB(r, g, b) ((r << 16) | (g << 8) | b)
```

Comment:
Or what about:

```c
static inline uint32_t rgb(uint8_t r, uint8_t g, uint8_t b) {
  return (r << 16) | (g << 8) | b;
}
```

Assuming that we're always supplying and expecting those types. Didn't look it up (for which my apologies).